doc/howto/network_increase_bandwidth.md

```diff
@@ -41,4 +41,8 @@ Complete the following steps to increase the network bandwidth on the LXD host:
 
 ## Increase the transmit queue length on the instances
 
-You must also change the `txqueuelen` value for all Ethernet interfaces in your instances, in the same way as described for the LXD host.
+You must also change the `txqueuelen` value for all Ethernet interfaces in your instances.
+There are two alternative ways to do this:
+
+- apply the same changes as described above for the LXD host, or
```

Comment:
@ru-fu do we need a capital for first letter here?

Reply:
Yes, and we shouldn't use an "or" at the end of a list item. I added a suggestion. :)